bin/respirate

```diff
@@ -5,6 +5,11 @@ require_relative "../loader"
 
 d = Scheduling::Dispatcher.new
 
+if Config.heartbeat_url && Strand[prog: "Heartbeat"].nil?
+  puts "Starting heartbeat prog"
+  Strand.create_with_id(schedule: Time.now, prog: "Heartbeat", label: "wait")
```

Comment:
This creates new strand each time we restart, right? I think we should create the Strand manually once.